packages/widgets/Source/I3SBSLExplorer/I3SBSLExplorer.js

```diff
@@ -0,0 +1,78 @@
+import { defined, DeveloperError } from "@cesium/engine";
+import knockout from "../ThirdParty/knockout.js";
+import I3SBSLExplorerViewModel from "./I3SBSLExplorerViewModel.js";
+
+/**
+ * I3S Building Scene Layer widget
+ *
+ * @alias I3SBSLExplorer
+ * @constructor
+ *
+ * @param {string} containerId The DOM element ID that will contain the widget.
+ * @param {I3SDataProvider} i3sProvider I3S Data provider instance.
+ *
+ * @demo {@link https://sandcastle.cesium.com/index.html?src=I3S%20Building%20Scene%20Layer.html|I3S Building Scene Layer}
+ */
+function I3SBSLExplorer(containerId, i3sProvider) {
+  const container = document.getElementById(containerId);
+
+  if (!defined(container)) {
```

Comment:
Use `Check` instead of throwing the developer error directly.

Reply:
Done.